components/Alert.html

```diff
@@ -36,7 +36,91 @@
+                         To use just add a class like for primary alert use class <span class="comp__detail-text">.success__alert</span> along with <span class="comp__detail-text">.alert</span> and to add icon we are passing icon as chil element.
+                    </p> 
+                    <div class="component__view">
+                        <div class="alert success__alert">
+                            <p class="alert___text"> <span class="alert__icon"><i class='bx bx-check-circle'></i></span> This is a simple success alert--check it out</p>
+                        </div>
+                        <div class="alert danger__alert">
+                            <p class="alert___text"> <span class="alert__icon"><i class='bx bxs-skull'></i></span> This is a simple danger alert--check it out</p>
+                        </div>
+                        <div class="alert warning__alert">
+                            <p class="alert___text"> <span class="alert__icon"><i class='bx bxs-skull'></i></span> This is a simple warning alert--check it out</p>
+                        </div>
+                        <div class="alert info__alert">
+                            <p class="alert___text"> <span class="alert__icon"><i class='bx bx-info-circle'></i></span> This is a simple info alert--check it out</p>
+                        </div>
+                    </div> 
+                    <div class="code__snippet">
+                        <iframe
+                        src="https://carbon.now.sh/embed/HZDhYgzR3PZjAAa7JrYj"
+                        style="width: 1280px; height: 546px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+
+                    <p class="component__heading-primary">Alert with links and cross button</p>
+                    <p class="component__sub-text">
+                         Alerts with icons can also be used here are some examples - <span class="comp__detail-text">success alert, danger alert, warning alert, info alert.</span> 
+                         To use just add a class like for primary alert use class <span class="comp__detail-text">.success__alert</span> along with <span class="comp__detail-text">.alert</span> and to add icon we are passing icon as chil element.
+                    </p> 
+                    <div class="component__view">
+                        <div class="alert success__alert">
+                            <p class="alert___text"> This is a simple link alert -- <a href="#" class="alert__link">check it out</a> <span class="cross__icon"><i class='bx bx-x'></i></span></p>
+                        </div>
+                        <div class="alert danger__alert">
+                            <p class="alert___text"> This is a simple danger alert--check it out <span class="cross__icon"><i class='bx bx-x'></i></span> </p>
```

Comment:
in BEM convention we dont use  "__modifier" which means if you are making a variation of the element then do it like "--modifier" that is "alert--success", but in "p" tag its correct